spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/adapter/DelegatingInvocableHandler.java

```diff
@@ -40,20 +49,35 @@
  */
 public class DelegatingInvocableHandler {
 
+	private static final ThreadLocal<Address> defaultReplyTo = new ThreadLocal<Address>();
```

Comment:
I don't like this solution, but, yes: I can't find the other way to deal with the `replyTo`...